controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -43,6 +45,9 @@ const (
 	StorageConsumerAnnotation     = "ocs.openshift.io.storageconsumer"
 	StorageClaimAnnotation        = "ocs.openshift.io.storageclaim"
 	StorageCephUserTypeAnnotation = "ocs.openshift.io.cephusertype"
+	MaxPGPerOSD                   = 600
+	MaxPGUnitSize                 = 128
```

Comment:
Please do not duplicate these numbers/constants in two distinct places (it will just create a divergence in future updates), have a single place to query the values from.

Reply:
Done added to util/cephutil.go